src/routes/github/create-branch/github-create-branch-get.ts

```diff
@@ -26,11 +30,18 @@ export const GithubCreateBranchGet = async (req: Request, res: Response, next: N
 		return next(new Error(Errors.MISSING_ISSUE_KEY));
 	}
 
-	const branchSuffix = summary ? replaceSpaceWithHyphenHelper(summary as string) : "";
+	const subscriptions = await Subscription.getAllForHost(jiraHost);
+
+	// TODO - this should redirect to a you are not configured page instead.
```

Comment:
Will create a new story for this, bascially if they haven't configured we should kick them off to somewhere betters